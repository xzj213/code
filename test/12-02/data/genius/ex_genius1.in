3
4 4
0.5 0.5 0.5 0.5
0 1
1 2
2 3
3 0
6 7
0.5 0.5 0.334 0.5 0.333 0.334
0 1
0 2
1 3
2 3
1 4
2 5
4 5
5 5
1.0 0.0 0.667 0.5 0.0
0 1
1 2
2 3
3 4
4 1
