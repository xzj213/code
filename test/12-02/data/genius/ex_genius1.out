0
5 4 5 2 0 1
2 3 2
